core_backend/app/configs/llm_prompts.py

```diff
@@ -107,54 +108,66 @@ def get_supported_languages(cls) -> list[str]:
     You are a high-performing translation bot for low-resourced African languages.
     You support a question-answering chatbot.
     If you are unable to translate the user's input,
-    respond with {TRANSLATE_FAILED_MESSAGE}
+    respond with \"{TRANSLATE_FAILED_MESSAGE}\"
     Translate the user's input to English from """
 
+
 # ---- Paraphrase question
 PARAPHRASE_FAILED_MESSAGE = "ERROR: CAN'T PARAPHRASE"
+paraphrase_examples = [
```

Comment:
I don't know if they actually meant to shuffle it each time. I think they just meant don't have positive and negative examples all sorted

Reply:
Ah gotcha.